### PR TITLE
Replaced secio flag with insecure in echo example readme file

### PR DESCRIPTION
**Pull request by mortezaalizadeh**

### CODE REVIEW

File: echo/README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ The `makeBasicHost()` function creates a [go-libp2p-basichost](https://godoc.org
 
 In order to create the swarm (and a `basichost`), the example needs:
 
-- An [ipfs-procotol ID](https://godoc.org/github.com/libp2p/go-libp2p-peer#ID) like `QmNtX1cvrm2K6mQmMEaMxAuB4rTexhd87vpYVot4sEZzxc`. The example autogenerates a key pair on every run and uses an ID extracted from the public key (the hash of the public key). When using `-secio`, it uses the key pair to encrypt communications (otherwise, it leaves the connections unencrypted).
+- An [ipfs-procotol ID](https://godoc.org/github.com/libp2p/go-libp2p-peer#ID) like `QmNtX1cvrm2K6mQmMEaMxAuB4rTexhd87vpYVot4sEZzxc`. The example autogenerates a key pair on every run and uses an ID extracted from the public key (the hash of the public key). When using `-insecure`, it uses the key pair to encrypt communications (otherwise, it leaves the connections unencrypted).
```
Comment (marten-seemann) [inline echo/README.md]: It's the other way around.

File: echo/README.md [hunk 2]
```diff
@@ -21,9 +21,9 @@ From the `go-libp2p-examples` directory run the following:
 ## Usage
 
 ```
-> ./echo -secio -l 10000
+> ./echo -insecure -l 10000
```
Comment (marten-seemann) [inline echo/README.md]: We should remove the flag here.

File: echo/README.md [hunk 3]
```diff
@@ -21,9 +21,9 @@ From the `go-libp2p-examples` directory run the following:
...
 2017/03/15 14:11:32 I am /ip4/127.0.0.1/tcp/10000/ipfs/QmYo41GybvrXk8y8Xnm1P7pfA4YEXCpfnLyzgRPnNbG35e
-2017/03/15 14:11:32 Now run "./echo -l 10001 -d /ip4/127.0.0.1/tcp/10000/ipfs/QmYo41GybvrXk8y8Xnm1P7pfA4YEXCpfnLyzgRPnNbG35e -secio" on a different terminal
+2017/03/15 14:11:32 Now run "./echo -l 10001 -d /ip4/127.0.0.1/tcp/10000/ipfs/QmYo41GybvrXk8y8Xnm1P7pfA4YEXCpfnLyzgRPnNbG35e -insecure" on a different terminal
```
Comment (marten-seemann) [inline echo/README.md]: Same here.

File: echo/README.md [hunk 4]
```diff
@@ -32,7 +32,7 @@ The listener libp2p host will print its `Multiaddress`, which indicates how it c
 Now, launch another node that talks to the listener:
 
 ```
-> ./echo -secio -l 10001 -d /ip4/127.0.0.1/tcp/10000/ipfs/QmYo41GybvrXk8y8Xnm1P7pfA4YEXCpfnLyzgRPnNbG35e
+> ./echo -insecure -l 10001 -d /ip4/127.0.0.1/tcp/10000/ipfs/QmYo41GybvrXk8y8Xnm1P7pfA4YEXCpfnLyzgRPnNbG35e
```
Comment (marten-seemann) [inline echo/README.md]: And here.

**Comment by mortezaalizadeh** [comment]
Sorry, my bad. just blindly did a find/replace. It should be all good to merge now.

**Pull request merged**